tests/python/contrib/test_dnnl.py

```diff
@@ -574,7 +580,6 @@ def get_graph(op, x_shape=(1, 8, 3, 3)):
         relay.log,
         relay.sqrt,
         relay.round,
-        relay.logsumexp,

```

Comment:
seems not relates to this PR, is it intended?

Reply:
yes. I remove this op, because I find it has not been supported completely.